wagtail/admin/templates/wagtailadmin/pages/index.html

```diff
@@ -9,6 +9,8 @@
     <div id="listing-results">
         {% include 'wagtailadmin/pages/index_results.html' %}
     </div>
+    {% trans "Select all pages in listing" as select_all_text %}
+    {% include 'wagtailadmin/bulk_actions/footer.html' with select_all_obj_text=select_all_text app_label='wagtailcore' model_name='page' objects=pages %}
```

Comment:
All the other views that use bulk actions include this template in the index view, not the AJAX results template. Putting it in the AJAX results will mean that we need to rebind the action buttons on search success, not just the checkboxes.

There's another issue with this and the "Select all items" button that seems to never have worked in the other views due to a logic error in the JS, but I think that's for another time.

Reply:
Okay, I think putting the `footer.html` include in the results template is the right way to go in the future, but:
- We'll need to rebind the event listeners for the action buttons as well, not just the checkboxes. This will be much easier if we have already migrated `bulk-actions.js` to Stimulus, as the rebinding will happen automatically.
- We'll need to update all the other templates for views that have bulk actions with search, i.e. snippets, images, documents, users, as they currently have the same bug.
- Make sure that when using the "Select all pages in listing" feature (which results in `id=all`), the bulk actions views should respect the search query result, which may include further descendants. Currently the bulk actions only consider direct children. I tried fixing it in 45cd52a.

If we put the include in the full index template instead, then the "Select all pages in listing" detection based on the pagination will not correctly reflect the search results (it will only be based on the initial render). Alternatively we can also detect the pagination in the JS...